certbot-apache/tests/entrypoint_test.py

```diff
@@ -9,40 +8,34 @@
 from certbot_apache._internal import entrypoint
 
 
-class EntryPointTest(unittest.TestCase):
-    """Entrypoint tests"""
-
-    _multiprocess_can_split_ = True
```

Comment:
Was this just for unittest?

Reply:
Even better! It was [for `nose`](https://nose.readthedocs.io/en/latest/doc_tests/test_multiprocess/multiprocess.html#controlling-distribution) which [we stopped using in 2017](https://github.com/certbot/certbot/pull/5282)! It looks like I removed them all as part of that PR, but a few have crept back in...

I opened https://github.com/certbot/certbot/pull/9603 to remove the rest of them.

Reply:
amazing.